boards/arm64/fvp-v8r/fvp-armv8r/configs/nsh_smp/defconfig

```diff
@@ -14,6 +14,8 @@ CONFIG_ARCH_CHIP_FVP_ARMV8R=y
 CONFIG_ARCH_CHIP_FVP_R82=y
 CONFIG_ARCH_EARLY_PRINT=y
 CONFIG_ARCH_INTERRUPTSTACK=4096
+CONFIG_ARM64_SMP_BUSY_WAIT=y
+CONFIG_ARM64_SMP_BUSY_WAIT_FLAG_ADDR=393216
```

Comment:
let's use hex number by changing CONFIG_ARM64_SMP_BUSY_WAIT_FLAG_ADDR type to hex